core/commands/wallet.go

```diff
@@ -173,3 +180,136 @@ var walletBalanceCmd = &cmds.Command{
 	},
 	Type: MessageOutput{},
 }
+
+var walletPasswordCmd = &cmds.Command{
+	Helptext: cmds.HelpText{
+		Tagline:          "BTFS wallet password",
+		ShortDescription: "set password for BTFS wallet",
+	},
+	Arguments: []cmds.Argument{
+		cmds.StringArg("password", true, false, "password of BTFS wallet."),
+	},
+	Options: []cmds.Option{},
+	Run: func(req *cmds.Request, res cmds.ResponseEmitter, env cmds.Environment) error {
+		cfg, err := cmdenv.GetConfig(env)
```

Comment:
This needs to be protected by a new wallet init flag (under config.UI section), if already init, cannot set password again.